### PR TITLE
build NLP model to predict NGAP codes from treatments found by spaCy …

### PR DESCRIPTION
**Pull request by GeoffroyGit**
Build NLP model to predict NGAP codes from treatments found by spaCy model.
I tested the model our our training data (with train set, validation set and test set): the accuracy is 0.7

### CODE REVIEW

**Comment by JulianBreaud** [comment]
OK
